recipes-rubygems/rubygems-nokogiri_1.12.3.bb

```diff
@@ -4,12 +4,13 @@ DESCRIPTION = "Nokogiri (鋸) makes it easy and painless to work with XML and HT
 HOMEPAGE = "https://nokogiri.org"
 
 LICENSE = "MIT"
```

Comment:
potential issue: Variable 'DESCRIPTION' is set by rubygems-nokogiri_1.12.3.bb,rubygems-nokogiri_1.11.5.bb